content/docs/user-guide/components/reference/vegetation-filters/vegetation-altitude-filter.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Vegetation Altitude Filter Component
+linktitle: Vegetation Altitude Filter
+description: Use the Vegetation Altitude Filter component to limit vegetation to a range of heights in your Open 3D Engine (O3DE) level.
+weight: 100
+---
+
+Add the **Vegetation Altitude Filter** component to limit vegetation or blocker instance placement to a specific height range.
+
+## Provider
+
+[Vegetation Gem](/docs/user-guide/gems/reference/environment/vegetation/)
+
+## Dependencies
+
+Add one of the following required components when using the Vegetation Altitude Filter component:
+- [**Vegetation Layer Blender**](./../vegetation/vegetation-layer-blender)
+- [**Vegetation Layer Blocker**](./../vegetation/vegetation-layer-blocker)
+- [**Vegetation Layer Blocker (Mesh)**](./../vegetation/vegetation-layer-blocker-mesh)
+- [**Vegetation Layer Spawner**](./../vegetation/layer-spawner)
```

Comment:
We don't typically use bold formatting on links. Is there a reason why these links are bolded, while the **Vegetation Gem** provider link above these is not? FYI, this inconsistency appears in several topics in this PR. I'll point out a few more, but not necessarily all of the others.

Reply:
I am bolding them because they are the first occurrence of a component name in the page, I haven't been doing the same for gems. I think it looks clunky, in later component ref PRs I believe I end up dropping bolding from bulleted lists at the very least.  I'd be for reserving bolding for sections of prose (for example the first line of each page) and removing it from the formulaic Dependencies, Properties, EBus sections.  I can't speak to bolding in relation to links, I may need a course-correction there.

Reply:
OK, I'm fine with leaving the bolding here for the reason indicated. I think we should revisit the "bolding first occurrences" guideline though, because - as you stated - it can get clunky.